python/benchmark/chemistry/chemistry_benchmark.py

```diff
@@ -36,13 +36,18 @@ def execute(self, inputParams):
         elif inputParams['Observable']['name'] == 'fermion':
             obs_str = inputParams['Observable']['obs_str']
             H = xacc.getObservable('fermion', obs_str)
-
         elif inputParams['Observable']['name'] == 'psi4':
```

Comment:
These 2 elifs provide the same code, you should remove the pyscf one and replace the one elif with 
```python
elif inputParams['Observable']['name'] in ['pyscf','psi4']:
```

Reply:
I fixed this - Do I need to make a new pull request in order for the new changes to take affect?